src/main/java/org/ssor/boss/core/transfer/RegisterUserInput.java

```diff
@@ -13,31 +13,31 @@
 /**
  * A data transfer object for the input of registering a new user.
  * <p>
- *   Registering a user takes place at the website public portal. A potential user enters their, to be made, profile
- *   information and clicks submit. A request is made to the gateway at <code>/api/v1/users</code> with a
- *   {@link RegisterUserInput} instance. The input is sent to the controller for users and then to the user service
- *   class where it will be processed and a new user will be registered. When successful, a {@link RegisterUserOutput}
- *   will be sent back through the gateway to the client.
+ * Registering a user takes place at the website public portal. A potential user
+ * enters their, to be made, profile information and clicks submit. A request is
+ * made to the gateway at <code>/api/v1/users</code> with a
+ * {@link RegisterUserInput} instance. The input is sent to the controller for
+ * users and then to the user service class where it will be processed and a new
+ * user will be registered. When successful, a {@link RegisterUserOutput} will
+ * be sent back through the gateway to the client.
  * </p>
  */
 @Data
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class RegisterUserInput implements Serializable
-{
-  @Size(min = 2, max = 16)
-  @Pattern(regexp = "^[a-zA-Z]+([_ -]?[a-zA-Z0-9])*$", message = "Please provide a valid username")
-  @NotBlank(message = "Please provide a valid username")
-  private String username;
+public class RegisterUserInput implements Serializable {
```

Comment:
@christianangeles Here as well.
```suggestion
public class RegisterUserInput implements Serializable 
{
```